METHOD_NAMING_PROPOSAL.md

```diff
@@ -0,0 +1,412 @@
+
+### Minimal Change
+
+With this implementation, the method names from v4 are maintained where possible. The comments above each method show what the method actually does, as the naming doesn't entirely align with this approach.
+
+<table>
+<tr>
+<th>v4 PayPal</th>
+<th>v5 PayPal</th>
+</tr>
+<tr>
+
+<td>
+
+```kotlin 
+// Retrieve data required to launch browser
+// Launch browser flow, parse browser result
+// Return nonce to listener
+payPalClient.tokenizePayPalAccount(activity, request)
+```
+
+</td>
+
+<td>
+
+```kotlin
+payPalLauncher = PayPalLauncher() { payPalBrowserSwitchResult ->
+  // Parse the result from the PayPal browser flow and return a nonce
+  payPalClient.onBrowserSwitchResult(payPalBrowserSwitchResult) 
+}
+
+// Retrieve data required to launch the PayPal flow in a browser
+payPalClient.tokenizePayPalAccount(context, request) { payPalResponse, error ->
+  // Launch the PayPal in a browser
+  payPalLauncher.launch(activity, payPalResponse) 
+}
+```
+
+</td>
+
+</tr>
+
+<tr>
+<th>v4 Local Payment</th>
+<th>v5 Local Payment</th>
+</tr>
+<tr>
+
+<td>
+
+```kotlin 
+// Retrieve data required to start local payment authorization
+// Or return result if no additional auth required
+localPaymentClient.startPayment(activity, request) { localPaymentResult, error ->
+  // Initiate browser flow and return nonce to listener
+  localPaymentClient.approveLocalPayment(activity, localPaymentResult)
+}
+```
+
+</td>
+
+<td>
+
+```kotlin
+localPaymentLauncher = LocalPaymentLauncher() { localPaymentBrowserSwitchResult ->
+  // Parse the result from the local payment browser flow and return a nonce
+  localPaymentClient.approveLocalPayment(localPaymentBrowserSwitchResult) 
+}
+
+// Retrieve data required to launch the local payment flow in a browser
+localPaymentClient.startPayment(context, request) { localPaymentResult, error ->
+  // Launch the local payment in a browser
+  localPaymentLauncher.launch(activity, localPaymentResult) 
+}
+```
+
+</td>
+
+</tr>
+<tr>
+<th>v4 3DS</th>
+<th>v5 3DS</th>
+</tr>
+<tr>
+
+<td>
+
+```kotlin 
+// Retrieve data required to start 3DS authentication
+// Or return result if no additional auth required
+threeDSecureClient.performVerification(activity, request) { threeDSecureResult, error ->
+  // Launch 3DS flow in Cardinal SDK 
+  // Return nonce to listener
+  threeDSecureClient.continuePerformVerification(activity, request, threeDSecureResult)
+}
+```
+
+</td>
+
+<td>
+
+```kotlin
+threeDSecureLauncher = ThreeDSecureLauncher() { authenticationResult ->
```

Comment:
it's been awhile but we may want to confirm if this is a result from the authentication result of 3DS or lookup result

Reply:
Pretty sure this would be the lookupResult. And the merchant can decide here wether or not to show the challenge or not. If they decide to take on the liability themselves, they would skip the `threeDSecureClient.continuePerformVerification()` call and directly call `cardClient.tokenize()` if they choose